src/hooks/analytics/useTrackPreviouslyCreatedAccounts.ts

```diff
@@ -0,0 +1,46 @@
+import { useStoredAccounts } from '@recoil/accounts';
+import { useSetting } from '@recoil/settings';
+import { useAnalyticsStatus } from '@recoil/analytics';
+import useTrackNewAccountAdded from 'hooks/analytics/useTrackNewAccountAdded';
+import React from 'react';
+
+/**
+ * Hook to track previously created accounts, this is required in such cases
+ * where the user have imported or created accounts before the analytics
+ * feature was added.
+ */
+const useTrackPreviouslyCreatedAccounts = () => {
```

Comment:
@bragaz @RiccardoM  Since we have already some installations this will allow us to track what the user have imported before adding PostHog. Note since I don't know how if those accounts was imported or created all those will be treated as imported.